packages/vector_graphics/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.16
+
+* Fixes some memory leaks.
```

Comment:
Please add a more descriptive changelog entry.

Reply:
Sure, what do you think of [More description changelog](https://github.com/flutter/packages/pull/8373/commits/f1a0d2165c007382854f1ee35e7161c9c6ac3897)?